frontends/web/src/components/rates/rates.tsx

```diff
@@ -56,11 +46,6 @@ apiGet('config').then((appconf) => {
   }
 });
 
-// TODO: should not invoking apiGet imediatelly, see the apiGet() function for more details
-apiGet('rates').then(rates => store.setState({ rates }));
```

Comment:
remove the backend /rates handler and the backend 'rates' push notifications? they now seem unused

Reply:
I left that because I am not sure if it could be useful for addressing the issue of transaction rates, which are not updating in send view. I can create the issue saying that there is this problem and that we should remove this code if it is still unused after fixing it. wdyt? Just to avoid deleting and then rewriting the same code